src/Services/membersService.js

```diff
@@ -1,10 +1,8 @@
 import axios from 'axios';
 
-const MEMBERS_URL = 'http://ongapi.alkemy.org/api/members';
```

Comment:
if all the urls are the same create only one url and replace it this const instead of on all the methods